src/useChildWindow.ts

```diff
@@ -30,8 +30,11 @@ export default ({
 
     const inheritCss = useCallback(() => {
         if (parentDocument && htmlDocument) {
-            const parentStyles = parentDocument.getElementsByTagName("style");
-            injectNodes(parentStyles, htmlDocument);
```

Comment:
If the parent window contained both inline `<style></style>` tags and external css, would we want both of these to be applied to the Child Window? If not then this looks good.

Reply:
The use of `.stylesheets` returns an array of the inline `<style>` tags in the `<link>` tags so this should include both types of styles at once.

Reply:
Ah cool - no worries then